public/app/plugins/panel/geomap/view.ts

```diff
@@ -24,36 +24,78 @@ export const centerPointRegistry = new Registry<MapCenterItems>(() => [
     lat: 0,
     lon: 0,
   },
+  {
+    id: MapCenterID.Coordinates as string,
+    name: 'Coordinates',
+  },
   {
     id: 'north-america',
     name: 'North America',
     lat: 40,
     lon: -100,
     zoom: 4,
   },
+  {
+    id: 'south-america',
+    name: 'South America',
+    lat: -20,
+    lon: -60,
+    zoom: 3,
+  },
   {
     id: 'europe',
     name: 'Europe',
     lat: 46,
     lon: 14,
     zoom: 4,
   },
+  {
+    id: 'africa',
+    name: 'Africa',
+    lat: 0,
+    lon: 30,
+    zoom: 3,
+  },
   {
     id: 'west-asia',
     name: 'West Asia',
     lat: 26,
     lon: 53,
     zoom: 4,
   },
+  {
+    id: 's-asia',
+    name: 'South Asia',
+    lat: 19.5,
+    lon: 80,
+    zoom: 4.5,
+  },
   {
     id: 'se-asia',
-    name: 'South-east Asia',
+    name: 'South-East Asia',
     lat: 10,
     lon: 106,
     zoom: 4,
   },
   {
-    id: MapCenterID.Coordinates as string,
-    name: 'Coordinates',
+    id: 'e-asia',
+    name: 'East Asia',
+    lat: 33,
+    lon: 120,
+    zoom: 4.5,
+  },
+  {
+    id: 'australia',
+    name: 'Australia',
+    lat: -25,
+    lon: 135,
+    zoom: 4.5,
```

Comment:
I would change this to a zoom of 4 to ensure the region of interest fits regardless of panel aspect ratio.